modules/base/humanitec.tf

```diff
@@ -30,21 +30,6 @@ resource "humanitec_resource_definition_criteria" "k8s_cluster_driver" {
   env_type               = var.environment
 }
 
-
-resource "humanitec_resource_definition" "k8s_logging" {
```

Comment:
This will also require a `make docs` to update the auto-generated terraform docs.

Reply:
Done, sorry for the delays.